### DISCUSSION TITLE
How to convert files from ED64 for use with OSCR?

### DISCUSSION THREAD

**Discussion by ameturpoet** [sanni/cartreader#834] [Need help]
I'm trying to write save files from my Everdrive 64, but whenever it finishes writing, it always seems to fail the verification step, and no change seems to have taken place with the save file when I boot them up in my N64. Do I need to do some kind of conversion to make the saves compatible? I've tested with Conker's Bad Fur Day and Kirby 64, both saves in .eep format.

**Comment by Ancyker** [comment]
Just checking, but you mean save files from the Everdrive to an original cartridge, right?

**Reply by ameturpoet** [reply]
Correct, I'm hoping to write them back to my real cartridges.

**Reply by Ancyker** [reply]
Try this: https://github.com/sanni/cartreader/discussions/593

**Reply by ameturpoet** [reply]
Hmm, I think that I've got the right filetype on both sides - the Everdrive and the OSCR both seem to use .eep and .fla saves instead of Project 64's .srm files. I think that there's just something preventing the OSCR from actually writing the save back to the cartridge, since it always fails verification after writing the save, and the cartridge's save always remains unchanged afterwards. Is there something else that could be the issue? Maybe there's something wrong with my OSCR hardware itself?

![IMG_6573](https://github.com/sanni/cartreader/assets/8140378/6a72ea23-4c39-4107-bbfc-d68928a7089d)

**Reply by ameturpoet** [reply]
I just did a test where I dumped my save to the SD card, then progressed a bit on thee cartridge so the save updated. Then I went back to the OSCR and wrote the save to the cartridge. This time there was no error, but the save file still did not change, and still showed the progress that I should have just overwritten.

**Reply by ameturpoet** [reply]
I'm also seeing the same behavior for GBA games - the write says it's successful, but nothing updates on the cartridges. What could be going on? I think I'll make a different topic about this since the issue seems to have changed compared to my initial guess.
